apps/database/cloudformation.yml

```
@@ -36,16 +56,115 @@ Resources:
       PubliclyAccessible: true
       # TODO: StorageType: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html#cfn-rds-dbinstance-storagetype
 
+  DatabaseSubnetGroup:
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: !Sub "Subnet Group for ${AWS::StackName} database"
+      SubnetIds: !Ref PublicSubnetIds
+
   DatabaseSecurityGroup:
-    Type: AWS::RDS::DBSecurityGroup
+    Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: !Sub "Security group for ${AWS::StackName} database"
-      DBSecurityGroupIngress:
-        - EC2SecurityGroupId: !Ref ClientSecurityGroup
-        - CIDRIP: !Ref CidrIp
+      GroupDescription: !Sub "Security group for clients of ${AWS::StackName} database"
```

I think the previous wording was correct since this isn't the client security group.
```suggestion
      GroupDescription: !Sub "Security group for ${AWS::StackName} database"
```